src/components/Article.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+/* const Article = () => (
+  <h1>1111</h1>
+);
+
+export default Article; */
+export function Article(article) {
+  return (
+    <>
+      <h1>{`${article.title}`}</h1>
```

Comment:
redundant interpolation, fix it everywhere
```suggestion
      <h1>{article.title}</h1>
```